Cargo.lock

```diff
@@ -4643,9 +4644,9 @@ dependencies = [
 
 [[package]]
 name = "rustfix"
-version = "0.6.1"
+version = "0.6.0"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "ecd2853d9e26988467753bd9912c3a126f642d05d229a4b53f5752ee36c56481"
+checksum = "6f0be05fc0675ef4f47119dc39cfc46636bb77d4fc4ef1bd851b9c3f7697f32a"
```

Comment:
This and sysinfo looks unrelated.

Reply:
@TaKO8Ki if this change and the one below are removed, does rustc still compile? If so, lets avoid not completely necessary changes :)